test_config.yaml

```diff
@@ -24,6 +24,16 @@ three_play_api_transcript_url: https://static.3playmedia.com/
 # a token identifying a valid request from transcript provider
 transcript_provider_request_token: 1234a5a67cr890
 
+# ---
+# Celery Info
+# ---
+onsite_worker: False
+celery_threads: 1
+celery_app_name: veda_test_app
+celery_worker_queue: test_encode_worker
+celery_deliver_queue: test_deliver_worker
+celery_heal_queue: test_heal_queue
```

Comment:
we don't need these settings in this file.

Reply:
@muhammad-ammar Yep, I see above.